manager/integration/tests/test_basic.py

```diff
@@ -563,7 +564,7 @@ def backup_status_for_unavailable_replicas_test(client, volume_name,  # NOQA
     for r in volume.replicas:
         if r.name == replica_name:
             node = client.by_id_node(r.hostId)
-            node = client.update(node, allowScheduling=False)
+            node = set_node_scheduling(client, node, allowScheduling=False)
```

Comment:
Because the tests here are special to change disks frequently, I would suggest not blindly making related util functions just retrying but rather using a specific parameter to instruct we do this on purpose like below. 

```python
set_node_scheduling(client, node, allowScheduling=False, retry=True)
```
This should apply to other node/disk util functions.

Reply:
okay

Reply:
Addressed. By default, the `retry` is `False`. `retry` is `True` in test_node.py test cases and cleanup disks functions in common.py.